bambi/models.py

```diff
@@ -754,7 +754,7 @@ def prior_predictive(self, draws=500, var_names=None, omit_offsets=True, random_
         return idata
 
     # pylint: disable=protected-access
-    def predict(self, idata, kind="mean", data=None, draws=None, inplace=True):
+    def predict(self, idata, kind="mean", data=None, with_group_specific=True, draws=None, inplace=True):
```

Comment:
Why you added it between `data` and `draws`? To me the most natural is to add it as the last argument, just in case someone has code that matches arguments positionally.

Is there any other reason?

Reply:
OK, shifted it to the end. It felt natural to go there after specifying `kind` and `data` but agreed this could break compatibility if using positional args.